LondonBritishLibrary/orient/BLorient758.xml

```diff
@@ -31,22 +31,21 @@ schematypens="http://relaxng.org/ns/structure/1.0"
                         </altIdentifier>
                         <altIdentifier>
                             <idno>Wright 332</idno>
-                            
+
                         </altIdentifier>
-                        
+
                     </msIdentifier>
                     <msContents>
                         <summary/>
-                        
-                        
-                        
+                        <msItem xml:id="ms_i1">
+                          <title type="complete" ref="LIT1948Mashaf"></title>
+                          <textLang mainLang="gez"></textLang>
+                          <colophon xml:id="coloph1" xml:lang="gez"><locus from="143a"></locus>
+                          <seg part="I">ዛቲ፡ መጽሐፍ፡ <add>ዘአባ፡ ወልደ፡ ስምዖን፡</add> ዘተዓገላ፡ ወዘሠረቃ፡</seg>
+                          <note>The colophon has been tampered with.</note>
```

Comment:
"tampered with" sounds a bit colloquial to me.. Do you think he refers to the addition of the new owner's name? Maybe "has been modified later" or something similar?